### ISSUE TITLE
Way to get more specific details about scrubbed reads

### ISSUE THREAD

**Issue by jwhitney31337** [thegenemyers/DASCRUBBER#16]
Using "REPtrim <unscrubbed.db>" I can get statistics about how many reads have adaptamers, how many were trimmed at each end, etc.  
But is there a way to get information about how each/any _particular_ read has been marked?  I.e. if I want to create a slide showing an example of a read with a chimeric region scrubbed, an adaptamer scrubbed, a 5' trim, etc, is there a way to identify such?
I know I can see the mapping of unscrubbed->scrubbed read regions using DASmap, but here I am looking for information about the trimmed/discarded regions of the original (unscrubbed) reads.

Thanks in advance,

Joe

**Comment by thegenemyers** [comment]
Yes, there is, but its  a bit under the radar.  When  you run DAStrim it 
produces a number of  mask tracks
(such as produced for repeat masking or by say DBdust), that mark the 
various intervals that are Low Q,
Adapter sites, etc.  The information in these tracks can then displayed 
for a given read with DBshow, or
better yet, use DaViewer to actually look at the read piles and their 
annotations.

The above is a bit terse but hopefully steers you in the right 
direction.  I should probably document these
tracks properly and I'll try to get around to it soon.  In the meantime, 
let me know if you need more.

    -- Gene
